client/asset/btc/walletclient.go

```diff
@@ -203,7 +203,8 @@ func (wc *walletClient) SendToAddress(address string, value, feeRate uint64, sub
 	}
 	var txid string
 	// Last boolean argument is to subtract the fee from the amount.
-	err = wc.call(methodSendToAddress, anylist{address, float64(value) / 1e8, "dcrdex", "", subtract}, &txid)
+	btc := btcutil.Amount(value).ToBTC()
```

Comment:
Can I ask for a different variable name? `btc` is used a lot for something else in this package.